src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/versions/versions.ts

```diff
@@ -0,0 +1,26 @@
+import { t } from '@lingui/macro'
+import { UpgradeController } from './upgradeForms/JBController3_0_1'
+import { UpgradeFundingCycle } from './upgradeForms/V3/UpgradeFundingCycle'
+
+export type JBVersion =
+  | '3' // upgrades to layer 2 contracts
+  | 'JBController3_0_1' // upgrades to a new JBController
+
+export const VERSIONS: {
+  [k in JBVersion]: {
+    name: string
+    description?: () => string | JSX.Element
+    component: () => JSX.Element | null
+  }
+} = {
+  '3': {
+    name: 'Juicebox v3',
+    component: UpgradeFundingCycle,
+  },
+  JBController3_0_1: {
```

Comment:
**NOTE** for new upgrades we want to add, the dev will have to add a property to this object